src/etc/gdb_lookup.py

```diff
@@ -5,7 +5,6 @@
 from rust_types import *
 
 
-rust_enabled = 'set language rust' in gdb.execute('complete set language ru', to_string=True)
```

Comment:
I wonder why this check exists. @ortem, I think you added this code a while ago. Do you know why the `&str` pretty printer is disabled for Rust-aware GDB versions? Maybe because GDB can handle them natively?

Reply:
Rust-aware GDB support pretty-printing `&str` type, but not `&mut str` type. Removing this check allows me to handle both types.

Reply:
OK, that's fine then.